src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -76,15 +72,12 @@ public String getDescription() {
         return description;
     }
 
-    public boolean isDeleted() {
-        return isDeleted;
-    }
 
     /**
      * Returns a deleted form of the same recipe.
      */
     public Recipe delete() {
-        return new Recipe(id, ingredients, productId, productName, productQuantity, description, true);
+        return new Recipe(id, ingredients, productId, productName, productQuantity, description);
     }
```

Comment:
Is this used anywhere? It was deleted for the Item class, and UniqueRecipeList no longer uses it